schemas/context/opportunity.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "An XDM Business Opportunity.",
+  "definitions": {
+    "opportunity": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Opportunity ID",
+          "description": "Unique identifer of the opportunity",
+          "type": "string"
+        },
+        "xdm:type": {
+          "title": "Opportunity Type",
+          "description": "Type of Opportunity.",
+          "type": "string"
+        },
+        "xdm:sourceID": {
```

Comment:
Needs to be consistent with the semantic meaning across other schemas. Should be sourceType and be pushed to the crm-mixin.